Sources/BraintreeAmericanExpress/BTAmexRewardsBalanceRequest.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+/// The GET parameters for `v1/payment_methods/amex_rewards_balance`
+struct BTAmexRewardsBalanceRequest: Encodable {
+    
+    private let currencyIsoCode: String
```

Comment:
nit for a future PR but we should update this to `currencyISOCode` to match our style guide preferences. Same in the `init`